Babylon/groups/api/groups/connector/__init__.py

```diff
@@ -0,0 +1,24 @@
+import cosmotech_api
+from click import group, pass_context
+from click.core import Context
+from cosmotech_api.api.connector_api import ConnectorApi
+
+from .....utils.decorators import pass_api_configuration
```

Comment:
I prefer using `Babylon.utils.decorators` in this case but it's a matter of taste.